integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -814,7 +814,8 @@ public void callWebAppAndVerifyLoadBalancing(String webappName, boolean verifyLo
 
       LoggerHelper.getLocal().log(Level.INFO, "Curl cmd with response code " + curlCmdResCode);
       LoggerHelper.getLocal().log(Level.INFO, "Curl cmd " + curlCmd);
-
+      //check helm status
+      checkLoadBalancer();
```

Comment:
it's an ingress controller, not a load balancer. just a note - no need to change it